message.js

```diff
@@ -192,6 +198,7 @@ jspb.Message.GENERATE_FROM_OBJECT = goog.define(
  * @define {boolean} Whether to generate toString methods for objects. Turn
  *     this off if you do not use toString in your project and want to trim it
  *     from the compiled JS.
+ * @export
```

Comment:
this shouldn't need to be exported as it is only referenced in this file iirc

Reply:
removed